godot-google-play-billing/build.gradle

```diff
@@ -2,16 +2,16 @@ plugins {
     id 'com.android.library'
 }
 
-ext.pluginVersionCode = 2
-ext.pluginVersionName = "1.0.1"
+ext.pluginVersionCode = 3
+ext.pluginVersionName = "1.0.2"
 
 android {
-    compileSdkVersion 29
-    buildToolsVersion "29.0.3"
+    compileSdkVersion 30
+    buildToolsVersion "30.0.3"
 
     defaultConfig {
         minSdkVersion 18
-        targetSdkVersion 29 
+        targetSdkVersion 30
```

Comment:
The current `3.2` code still targets SDK 29, this won't be an issue? https://github.com/godotengine/godot/blob/3.2/platform/android/java/app/config.gradle#L1-L12

Reply:
I don't think it's a problem (the library with this PR applied works file with 3.2.4-beta4). IIRC only `minSdkVersion` is the critical part that has to match.

Reply:
https://stackoverflow.com/a/26694276/3691378

Reply:
I was thinking of potential changes in Google Play policy when targeting higher SDK versions, e.g. https://github.com/godotengine/godot/issues/38913

But this was for SDK 29, for SDK 30 it should be fine, the main issue we had so far is that signing must be done with `apksigner` instead of `jarsigner` but this is handled by https://github.com/godotengine/godot/pull/44645

So I guess it might be fine. Still, it might be safest to stay in sync with the Godot version which we're syncing with here.
CC @m4gr3d

Reply:
@akien-mga I agree, I'd not merge this PR until 3.2.4 stable anyways.

Reply:
I don't think we make use of storage access in this plugin (@timoschwarzer feel free to correct me if I'm wrong), so it should be safe to update this plugin to SDK 30.
As @timoschwarzer mentioned, only the `minSdkVersion` is the one we need to keep in sync  :) .